client/my-sites/email/email-providers-comparison/price/google-workspace.tsx

```diff
@@ -97,11 +99,8 @@ const GoogleWorkspacePrice = ( {
 
 	const canPurchaseGSuite = useSelector( canUserPurchaseGSuite );
 
-	if ( ! domain ) {
-		return <></>;
-	}
```

Comment:
Shouldn't we refine the condition instead of removing this whole clause?

```javascript
if ( ! domain && ! hasCartDomain ) {
  return <></>;
}
```

Reply:
That's a good point, let me study it

Reply:
Yes, I think we should refine it instead of removing it. I remember it was an early exit condition when domain wasn't loaded yet to avoid crashes. BUT it might be cases where even providing the domain without being in the cart, the domain hasn't loaded yet. Good catch!